### ISSUE TITLE
Error using joint in R2016b

### ISSUE THREAD

**Issue by 0xaf1f** [Craigacp/MIToolbox#3]
I compiled the matlab sources for MIToolbox using MATLAB R2016b and gcc 4.9.1, but I get the following error when running your examples to test:

```
>> y = [1 1 1 0 0]';
>> x = [1 0 1 1 0]';
>> joint([x,y])
Not enough input arguments.

Error in joint (line 12)
if (~isa(X,'double') || ~isa(arities,'double'))
```

The other examples worked fine. This is based on your latest release, 3.0.1.

**Comment by Craigacp** [comment]
Looks like I put that check in the wrong place. I've pushed out a commit to devel which fixes the issue.

**Comment by 0xaf1f** [comment]
Thanks--I can confirm that it now works for me as described. Would you mind tagging a new patch release?

**Comment by Craigacp** [comment]
Fixed in release v3.0.2, commit 94f87e8.
